### PR TITLE
[Doc] Adding Snap ops markdown

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
Added snap ops markdown doc.

Fixes: #839

Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

File: docs/BP/Ops/snap_ops.md [hunk 1]
```diff
@@ -0,0 +1,513 @@
...
+                - error_msg: error message
+                - error_code: error code returned
+                - cmd : command that got executed
+                - node : node on which the command got executed
+                
+        Example:
+        ```python
+            redant.enable_uss(self.vol_name, self.server_list[0])
+		```
+
```
Comment (aujjwal-redhat) [inline docs/BP/Ops/snap_ops.md:26]: It is not able to display in correct format.

Preview:

Screenshot from 2021-07-21 10-21-50

Comment (aujjwal-redhat) [reply docs/BP/Ops/snap_ops.md:26]: Same is the case in the ones below as well.

**Review by aujjwal-redhat — CHANGES_REQUESTED** [review]

File: docs/BP/Ops/snap_ops.md (hunk 1 above)
Comment (srijan-sivakumar) [reply docs/BP/Ops/snap_ops.md:26]: I think it is not working in markdown. I just checked across the docs. The example is sticking out...This means we might need to also clear out from other docs.

Comment (srijan-sivakumar) [reply docs/BP/Ops/snap_ops.md:26]: @aujjwal-redhat I think we can take this in for now. We need to decide whether we are going to live with this type of documentation of shifting to readthedocs or github wiki in the coming days. Till then, I think this won't cause any confusion.

Comment (aujjwal-redhat) [reply docs/BP/Ops/snap_ops.md:26]: We can remove the *```python* completely then else anyone will get confused on seeing the examples part. Just a suggestion else we can keep it as it is.

Comment (nik-redhat) [reply docs/BP/Ops/snap_ops.md:26]: Hmm, then that can be made but not an urgent requirement I would say.

Comment (aujjwal-redhat) [reply docs/BP/Ops/snap_ops.md:26]: Yeah.

**Review by aujjwal-redhat — APPROVED** [review]

**Pull request merged**